src/sql/workbench/contrib/editData/browser/editDataEditor.ts

```diff
@@ -292,11 +293,19 @@ export class EditDataEditor extends BaseEditor {
 			this._setSashDimension();
 
 			this._register(this._sash.onPositionChange(position => this._doLayout()));
+			this._register(this.themeService.onDidColorThemeChange(e => this.updateTheme(e)));
 		}
 
 		this._sash.show();
 	}
 
+	//update sash theme
```

Comment:
This should go away with the changes suggested above, but make sure that if you're leaving comments you leave valid XML doc comments